infrastructure/templates/public-api/components/alerts/storageAccounts/availabilityAlert.bicep

```diff
@@ -0,0 +1,33 @@
+import { Severity } from '../types.bicep'
+
+@description('Names of the resources that these alerts are being applied to.')
+param resourceNames string[]
+
+@description('The alert severity.')
+param severity Severity = 'Critical'
+
+@description('Name of the Alerts Group used to send alert messages.')
+param alertsGroupName string
+
+@description('Tags with which to tag the resource in Azure.')
+param tagValues object
+
+module alerts '../staticMetricAlert.bicep' = [for name in resourceNames: {
+  name: '${name}AvailabilityAlertModule'
+  params: {
+    alertName: '${name}-availability'
```

Comment:
No, this is because the ${name} part is actually the name of the storage account itself, so I figured it didn't need the additional `-storageaccount-` part.  The reason for the `-fileservice-` part here is because it's tagged onto the end of the storage account's name that it belongs to.